.github/workflows/ci.yml

```
@@ -1,168 +1,127 @@
+      - name: Install dependencies
+        run: npm install
+      - name: Build extension
+        run: npm run compile
+      - name: Run tests
+        run: npm test
+
+  lint-format:
+    name: Lint and Format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)